src/app/integrations/intacct/intacct-main/intacct-export-log/intacct-skip-export-log/intacct-skip-export-log.component.spec.ts

```
@@ -1,23 +1,143 @@
+    paginatorService = TestBed.inject(PaginatorService) as jasmine.SpyObj<PaginatorService>;
+    userService = TestBed.inject(UserService) as jasmine.SpyObj<UserService>;
+  });
 
+  beforeEach(() => {
     fixture = TestBed.createComponent(IntacctSkipExportLogComponent);
     component = fixture.componentInstance;
-    fixture.detectChanges();
+    userService.getUserProfile.and.returnValue({ org_id: 'ORG123' } as MinimalUser);
+    paginatorService.getPageSize.and.returnValue(mockPaginator);
+    exportLogService.getSkippedExpenses.and.returnValue(of(mockSkipExportLogResponse));
   });
```

Biome

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)